### PR TITLE
[ZH] Update `Rules`

### PR DESCRIPTION
**Pull request by FlyingFish23**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by RushFTK** [comment]
建议标题修改为如下形式，可参考其他翻译类PR的格式：
````
[ZH] Update `Rules`
````

**Comment by RushFTK** [comment]
Btw seemed you are editing same files compared with #7613 , have you merge these PRs?

**Comment by FlyingFish23** [comment]
第一次写osu!的wiki，其实除了markdown会用，别的我都不是很熟悉，GitHub怎么操作我也不清楚。几个月之前是改过一次，把页面最底部的48改成了英文原文的42，这一次把英文版新加的第六条放了进去。还需要我做什么可以直接说，内容可以直接更改，谢谢大家的悉心审核和意见。

**Comment by TicClick** [comment]
honest question, how does your work compare to https://github.com/ppy/osu-wiki/pull/8007, which covers the same area? it is generally preferable to work on already existing PRs by leaving reviews and other comments, unless they are poorly executed

**Comment by FlyingFish23** [comment]
I didn't saw that pr. Will take a look. If that one's better to work on then I'll close mine.


发自我的 Apple Watch

**Review by TPGPL — CHANGES_REQUESTED** [review]
- L43 seems outdated - missing a link at least
- L44 is missing a link
- L54 is missing a link
- L59 is missing a link
- L63 is missing a link

File: wiki/Rules/zh.md [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
-1. **一个玩家只能拥有*一个*帐号。从注册开始，** 你注册的*第一个*帐号是你的唯一的 osu! 账号。这个账号代表你。它不是其他任何人————你的父母，你的兄弟姐妹，你的朋友————它就是你。不要与其他任何人共享账号。如果你无法使用该账号，你也不应再创建一个新的账号。所以请保持你的账号安全。
+1. **一个玩家只能拥有*一个*帐号。从注册开始，** 你注册的*第一个*帐号是你的唯一的 osu! 账号。这个账号代表你。它不是其他任何人————你的父母，你的兄弟姐妹，你的朋友————它就是你。不要与其他任何人共享账号。账号失去即无，你无权创建新账号，所以请保持你的账号安全。
 2. **公平竞争。** 不允许使用任何第三方工具获取任何优势。包括宏脚本，辅助瞄准程序，调速工具等一切有利于你在游戏中操作的工具。任何替你做出游戏内本该你自己做出的动作的程序都不行。
-3. **待人和善。** 骚扰或敌视他人是不受欢迎的行为，请不要互相伤害。我们在这里点圈圈，敲鼓，接水果，或者同时敲最多八个键，而不是互相恶心。
+3. **待人和善。** 骚扰或敌视他人是不受欢迎的行为，请不要互相伤害。我们在这里点圈圈，敲鼓，接水果，或者同时敲最多十八个键，而不是互相恶心。
 4. **不要逃避。** 如果你不确定自己是否违反了规定，恐怕你已经违反了。
-5. **我们是全年龄社区。** 这意味着不能发布 R18/[NSFW](https://baike.baidu.com/item/NSFW/5576169)（Not Safe For Work，不宜在工作期间访问）内容，例如毒品，性。除非它们是为了配合语境，否则绝对不能使用。
-6. **规定中没有指出的，请根据常识做出判断。** 管理员有权给出他们认为合适的裁决。
+5. **我们是全年龄社区。** 这意味着不能发布 R18/[NSFW](https://zh.wikipedia.org/wiki/NSFW)（Not Safe For Work，不宜在工作期间访问）内容，例如毒品，性。除非它们是为了配合语境，否则绝对不能使用。
+6. **保持聊天室绿色健康** osu!不是约会或交友软件。对不愿意的用户使用露骨语言或图片或在公共或私聊中强行勾搭的行为，均不能被容忍。如果某人对你使用让你感到不自在的语言，请向我们[匿名举报](/wiki/Reporting_bad_behaviour/Abuse)，我们会保护你！这也能为其他用户净化社区环境，避免再次发生。
```
Comment (YumeMuzi) [inline wiki/Rules/zh.md:12]:
```suggestion
6. **保持聊天绿色健康** osu! 不是约会或交友软件。我们决不允许向不愿接收此类消息的用户发送露骨语言或图片，或是在聊天中强行勾搭的行为。如果某人向你发送了这一类冒犯性的言论，请向我们[匿名举报](/wiki/Reporting_bad_behaviour/Abuse)，我们会保护你！这也能避免这类事件再次发生，从而净化社区环境，保护其他用户。
```

File: wiki/Rules/zh.md [hunk 2]
```diff
@@ -1,20 +1,16 @@
...
 
 这些规章制度确保了 osu! 社区能对所有人都有一个有趣而友好的氛围。
 
 ## 社区规定
 
-1. **一个玩家只能拥有*一个*帐号。从注册开始，** 你注册的*第一个*帐号是你的唯一的 osu! 账号。这个账号代表你。它不是其他任何人————你的父母，你的兄弟姐妹，你的朋友————它就是你。不要与其他任何人共享账号。如果你无法使用该账号，你也不应再创建一个新的账号。所以请保持你的账号安全。
+1. **一个玩家只能拥有*一个*帐号。从注册开始，** 你注册的*第一个*帐号是你的唯一的 osu! 账号。这个账号代表你。它不是其他任何人————你的父母，你的兄弟姐妹，你的朋友————它就是你。不要与其他任何人共享账号。账号失去即无，你无权创建新账号，所以请保持你的账号安全。
 2. **公平竞争。** 不允许使用任何第三方工具获取任何优势。包括宏脚本，辅助瞄准程序，调速工具等一切有利于你在游戏中操作的工具。任何替你做出游戏内本该你自己做出的动作的程序都不行。
-3. **待人和善。** 骚扰或敌视他人是不受欢迎的行为，请不要互相伤害。我们在这里点圈圈，敲鼓，接水果，或者同时敲最多八个键，而不是互相恶心。
+3. **待人和善。** 骚扰或敌视他人是不受欢迎的行为，请不要互相伤害。我们在这里点圈圈，敲鼓，接水果，或者同时敲最多十八个键，而不是互相恶心。
```
Comment (YumeMuzi) [inline wiki/Rules/zh.md:9]:
```suggestion
3. **待人和善。** 骚扰或攻击他人是不受欢迎的行为，请不要互相伤害。我们在这里点圈圈，敲鼓，接水果，或者同时敲最多十八个键，而不是互相恶心。
```

File: wiki/Rules/zh.md [hunk 3]
```diff
@@ -1,20 +1,16 @@
...
----
-
 # 规章制度
 
 这些规章制度确保了 osu! 社区能对所有人都有一个有趣而友好的氛围。
 
 ## 社区规定
 
-1. **一个玩家只能拥有*一个*帐号。从注册开始，** 你注册的*第一个*帐号是你的唯一的 osu! 账号。这个账号代表你。它不是其他任何人————你的父母，你的兄弟姐妹，你的朋友————它就是你。不要与其他任何人共享账号。如果你无法使用该账号，你也不应再创建一个新的账号。所以请保持你的账号安全。
+1. **一个玩家只能拥有*一个*帐号。从注册开始，** 你注册的*第一个*帐号是你的唯一的 osu! 账号。这个账号代表你。它不是其他任何人————你的父母，你的兄弟姐妹，你的朋友————它就是你。不要与其他任何人共享账号。账号失去即无，你无权创建新账号，所以请保持你的账号安全。
```
Comment (YumeMuzi) [inline wiki/Rules/zh.md:7]: 这里不应该修改。因为这里英文原文是有大前提 `if you lose access to it` 的。

Comment (YumeMuzi) [inline wiki/Rules/zh.md:7]:
```suggestion
1. **一个玩家只能拥有*一个*帐号。从注册开始，** 你注册的*第一个*帐号是你的唯一的 osu! 账号。这个账号代表你。它不是其他任何人————你的父母，你的兄弟姐妹，你的朋友————它就是你。不要与其他任何人共享账号。即使你无法使用该账号，你也不应再创建一个新的账号。所以请保持你的账号安全。
```

**Review by YumeMuzi — CHANGES_REQUESTED** [review]
others seem fine

File: wiki/Rules/zh.md [hunk 4]
```diff
@@ -23,7 +19,7 @@ outdated_since: 2787f766bf4cf7b8e2e3afd824e15adcb4e79798
 3. **不要讨论被禁言或限制的玩家。** 他的问题只能由他自己和管理员解决，而不是其他人。
 4. **鼓励公平竞争，不要关注作弊玩家。** 很遗憾，作弊玩家确实存在，给予他们过多关注会鼓励他们继续或吸引他人模仿。请使用`!report`命令或是在个人资料页点击来举报他们。
 5. **语言得体，适合全年龄，避免 R18/NSFW 内容。** NSFW 内容并不适合全年龄社区，你可以在私聊中（合情合理地）畅所欲言，但请保持公共频道的清洁。
-6. **注意你与他人分享的内容，尤其是可能导致不适的内容。** 分享之前三思而后行。
+6. **注意你与他人分享的内容，尤其是可能导致不适的内容。** 如果在分享之前下意识的质疑了自己，那么这些内容可能真的不适合被分享。避免分享包含快速闪动灯光、血腥、滥用和容易引起不适的声像内容。
```
Comment (Moutreak) [inline wiki/Rules/zh.md:22]:
```suggestion
6. **注意你与他人分享的内容，尤其是可能导致不适的内容。** 如果在分享之前下意识地质疑了自己，那么这些内容可能真的不适合被分享。避免分享包含快速闪动灯光、血腥、滥用和容易引起不适的声像内容。
```

**Review by Moutreak — CHANGES_REQUESTED** [review]
副词与动词之间应该用“地”

**Comment by TicClick** [comment]
https://github.com/ppy/osu-wiki/pull/8033/files#diff-dc686f08c8c6d88718cf2868f4864087cbdeb95f64b43710546f10aa5e0aa557R43 this doesn't match the original rules (needs to mention content usage guidelines: https://github.com/ppy/osu-wiki/blob/a107694d0b85f15b70c540b2096fbb07d7d270d9/wiki/Rules/en.md?plain=1#L43). please fix this and we're good

File: wiki/Rules/zh.md [hunk 5]
```diff
@@ -45,7 +41,7 @@ outdated_since: 2787f766bf4cf7b8e2e3afd824e15adcb4e79798
 这些规定适用于通过游戏内的[谱面提交系统](/wiki/Submission)提交的谱面。当 osu! 客户端在[编辑器](/wiki/Client/Beatmap_editor)中提交谱面的适合会创建相应的讨论贴。
 
 1. **确保你拥有谱面内所有内容的使用权。** 包括歌曲、视频、击打声、图片以及其它不是你原创的内容。如果你不确定，我们为你提供了一个不断增长的[精选艺术家](https://osu.ppy.sh/beatmaps/artists)曲目库，它们全都被授权在 osu! 中使用。
```
Comment (YumeMuzi) [inline wiki/Rules/zh.md:43]:
```suggestion
1. **确保你拥有谱面内所有内容的使用权。** 包括歌曲、视频、击打声、图片以及其它不是你原创的内容。如果你不确定，我们为你提供了一个不断增长的[精选艺术家](https://osu.ppy.sh/beatmaps/artists)曲目库，它们全都被授权在 osu! 中使用。要了解更多信息，请参见：[内容使用规范](Content_Usage_Guidelines)。
```

**Comment by FlyingFish23** [comment]
all the links are there, has been done by other contributors i guess?

**Comment by FlyingFish23** [comment]
> https://github.com/ppy/osu-wiki/pull/8033/files#diff-dc686f08c8c6d88718cf2868f4864087cbdeb95f64b43710546f10aa5e0aa557R43 this doesn't match the original rules (needs to mention content usage guidelines:
> 
> https://github.com/ppy/osu-wiki/blob/a107694d0b85f15b70c540b2096fbb07d7d270d9/wiki/Rules/en.md?plain=1#L43
> 
> ). please fix this and we're good

thanks for adding the links!

**Review by TicClick — APPROVED** [review]

**Pull request merged**